src/Simplic.Flow.Service/FlowService.cs

```diff
@@ -107,7 +107,9 @@ public FlowService(IUnityContainer unityContainer)
             unityContainer.RegisterType<INodeResolver, GenericNodeResolver<ReadAllTextNode>>(nameof(ReadAllTextNode));
             unityContainer.RegisterType<INodeResolver, GenericNodeResolver<ClearPinNode>>(nameof(ClearPinNode));
 
-            flowConfigurations = flowConfigurationService.GetAll().ToList();
+            // Load active flow configurations
+            flowConfigurations = flowConfigurationService.GetAll().Where(x => x.IsActive).ToList();
```

Comment:
it would be better to put this extra method into the repository. 'GetAllActive()'